convert-hf-to-gguf.py

```diff
@@ -1186,7 +1186,6 @@ def write_tensors(self):
 @Model.register("StableLmForCausalLM", "StableLMEpochForCausalLM", "LlavaStableLMEpochForCausalLM")
 class StableLMModel(Model):
     model_arch = gguf.MODEL_ARCH.STABLELM
-
     def set_vocab(self):
```

Comment:
The flake8 linter says: `./convert-hf-to-gguf.py:1190:5: E301 expected 1 blank line, found 0`

```suggestion

    def set_vocab(self):
```